src/ui/DynamicHierarchicalFacet/DynamicHierarchicalFacetValues/DynamicHierarchicalFacetValues.ts

```diff
@@ -25,6 +25,7 @@ export class DynamicHierarchicalFacetValues implements IDynamicHierarchicalFacet
   }
 
   public createFromResponse(response: IFacetResponse) {
+    this._selectedPath = [];
```

Comment:
There is the `clearPath` method which could be used here to remove the duplication,

Reply:
Yeah!